_source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: blog_post
+title: "Developer Day 2021:  Authentication, Authorization, and Celebrating Our Diverse Developer Community"
+Author: ali-spivak
```

Comment:
```suggestion
author: ali-spivak
```

It's likely a case-sensitive issue and not the brackets (the brackets are only needed if you have more than one, but it shouldn't hurt either)